### ISSUE TITLE
[Discussion] get_col_position depth value testing in draw_ascii

### ISSUE THREAD

**Issue by xhlulu** [biopython/biopython#1910]
### Setup

CPython
Windows-10-10.0.17134
1.73

### Description

In `Phylo._utils`, we can use the `draw_ascii()` function to draw a command line-friendly representation of our phylogeny tree. The function can be found here:
https://github.com/biopython/biopython/blob/0593e2f4890897653c4cf6e0f76b9fb4adb5dfb2/Bio/Phylo/_utils.py#L206-L269

We notice that the `get_col_position` function assigns a unit branch length when there are no depth values, using the following conditional statement:
```python

"""Create a mapping of each clade to its column position."""
depths = tree.depths()
# If there are no branch lengths, assume unit branch lengths
if not max(depths.values()):
    depths = tree.depths(unit_branch_lengths=True)
```

However, I believe we could explicitize this condition by slightly modifying it:
```python
depths = tree.depths()
# If there are no branch lengths, assume unit branch lengths
if max(depths.values()) == 0:
    depths = tree.depths(unit_branch_lengths=True)
```
Which, although doesn't affect the behavior, makes the code clearer (since this method is particularly useful for people who want to learn how to draw phylogeny trees in different context). It also follows [PEP8 recommendations](https://www.python.org/dev/peps/pep-0008/#programming-recommendations).

**Comment by peterjc** [comment]
PEP8 doesn't seem to cover this, but in general Python style would be ``if expr:`` and ``if not expr:`` rather than ``if expr == 0`` and ``if expr != 0:`` when testing if something is or isn't zero.

However, I agree your proposal is clearer when reading the code. Over to @etal to decide.

**Comment by mkcor** [comment]
@peterjc Ok *in general* but, here, `max(depths.values())` is expected to be a number, so we might as well be explicit (instead of relying on the implicit type conversion), right?

In between `depths = tree.depths()` and the `if` statement, should we include some type checking or exception handling?

**Comment by xhlulu** [comment]
@peterjc Thanks for the heads up! I created the PR at https://github.com/biopython/biopython/pull/1916

**Comment by peterjc** [comment]
Closed via #1916.
